Makefile

```diff
@@ -23,6 +23,28 @@ PUSH_COMMAND = IMAGE_TAG=${IMAGE_TAG} .scripts/travis/push-image.sh
 
 DC_BUILD = IMAGE_TAG=${IMAGE_TAG} docker-compose -f docker-compose.build.yml
 
+TRAVIS_BRANCH ?= `git rev-parse --abbrev-ref HEAD -- | head -n 1`
+TRAVIS_PULL_REQUEST ?= false
+
+# TODO: Make this generic somehow
```

Comment:
using an external shell script? That could also be linted by `shellcheck`

Reply:
That'll work, I'll do that :thinking: